etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/utils/security/HashHelper.java

```diff
@@ -0,0 +1,25 @@
+package gov.hhs.cdc.trustedintermediary.etor.utils.security;
+
+import gov.hhs.cdc.trustedintermediary.wrappers.Logger;
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.HexFormat;
+import javax.inject.Inject;
+
+public class HashHelper implements SecureHash {
```

Comment:
I think this application could be a singleton class or be purely static.  For example, `public static String generateHash(...)`.  I'll add more to this a bit later.

Reply:
And if you are going to make this a singleton like class (like most of our other classes, e.g. `ApacheClient` and its `getInstance()` method), you could even register this with the `ApplicationContext` and use `@Inject` to bring it into `SendOrderUsecase` (and result one too).  In fact, I think I recommend this.